tests/pipelines/eks/awscli-eks-cl2-load.yaml

```diff
@@ -21,6 +21,9 @@ spec:
   - name: slack-message
   - name: vpc-cfn-url
   tasks:
+  - name: timestamp
```

Comment:
do we need timestamp as a separate task ?
wdyt about adding same timestamp as part of pushing the results here - https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/tests/tasks/generators/clusterloader/load.yaml#L91 ?

Reply:
I'm trying to use the following in the `TriggerTemplate`s:
```
- name: results-bucket
  value: kit-eks-scalability/kit-eks-1k/$(tasks.timestamp.results.timestamp)
```

Your approach works and simplify the wiring between tasks and triggertemplate.
But it changes the meaning of parameter `results-bucket`. It becomes to mean `results-bucket-prefix`. We will need to change the parameter name across the board.
I don't have a strong opinion here.

Reply:
Had an offline discussion with Harish. The reusability is not important for this use case, since task `load` is the only one relies on it. We are not going to create a separate task just for generating the timestamp.
I will push the new change shortly.

Reply:
> We have added the timestamp when passing the parameters in tekton `TriggerTemplate` hosted in the internal repo. We no longer need this PR. Closing



> We have added the timestamp when passing the parameters in tekton `TriggerTemplate` hosted in the internal repo. We no longer need this PR. Closing

Thanks David.